packages/toolkit/connector-kit/src/types.ts

```diff
@@ -162,13 +172,24 @@ export type SendMessageFunction = (
 export type SocialConnector = BaseConnector<ConnectorType.Social> & {
   getAuthorizationUri: GetAuthorizationUri;
   getUserInfo: GetUserInfo;
+  validateAssertion?: ValidateAssertion;
 };
 
+// This type definition is for SAML connector
+export type ValidateAssertion = (
```

Comment:
rename as SAMLValidateAssertion ?

Reply:
Renamed to ValidateSamlAssertion to follow v. + n. naming convention